File: 30-Days-Of-Code/Day 3/Intro to Conditional Statements.java
import java.util.Scanner;
public class hack{

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
            int n;
            n = scan.nextInt();
            
            //if even
            if(n%2 == 0)  
            {   
                if(n>=2&&n<=5)
                    System.out.println("Not Weird");
                else if(n>=6&&n<=20)
                	System.out.println("Weird");
                else if(n>20)
                	System.out.println("Not Weird");
            }
            
            //if odd
            else
            	System.out.println("Weird");
            
            scan.close();
        }
    }

File: 30-Days-Of-Code/Day 3/Intro to Conditional Statements.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    
    //if even
    if(n%2 == 0)  
    {   
        if(n>=2&&n<=5)
            cout<<"Not Weird";
        else if(n>=6&&n<=20)
            cout<<"Weird";
        else if(n>20)
            cout<<"Not Weird";
    }
    
    //if odd
    else
        cout<<"Weird";
    return 0;
}
